tests/contrib/utils/gcp_authenticator.py

```diff
@@ -31,7 +31,7 @@
 GCP_BIGTABLE_KEY = 'gcp_bigtable.json'
 GCP_SPANNER_KEY = 'gcp_spanner.json'
 GCP_GCS_KEY = 'gcp_gcs.json'
-GCP_AI_KEY = 'gcp_ai.json'
+GCP_AI_KEY = 'gcp_transfer.json'
```

Comment:
Why?

Reply:
Antoni did not have the keys locally on the computer, so we swapped it up quickly. After finishing work, he did not restore the change.